src/components/chromecast/chromecastplayer.js

```diff
@@ -574,8 +574,15 @@ define(['appSettings', 'userSettings', 'playbackManager', 'connectionManager', '
 
             events.trigger(instance, "playbackstop", [state]);
 
+            var state = instance.lastPlayerData.PlayState || {};
+            var volume = state.VolumeLevel;
```

Comment:
should we add a default like `|| 0.5`?.. and `false` to `mute`?

Reply:
yeah, makes sense.